cmd/operator-sdk/new/cmd.go

```diff
@@ -156,8 +156,10 @@ func doGoScaffold() error {
 	switch m := projutil.DepManagerType(depManager); m {
 	case projutil.DepManagerDep:
 		err = s.Execute(cfg, &scaffold.GopkgToml{})
+	case projutil.DepManagerGoMod:
+		err = s.Execute(cfg, &scaffold.GoMod{}, &scaffold.Tools{})
```

Comment:
@estroz Perhaps I'm having trouble with a user error but I can't create a new project outside of my `$GOPATH`:
```console
$ operator-sdk new memcached-operator
INFO[0000] Creating new Go operator 'memcached-operator'.
FATA[0000] Project not in $GOPATH
```

Looking at the above error, won't this be caused by [Line 149](https://github.com/operator-framework/operator-sdk/blob/01b6886ae9324212ca05b20a75cf98d0dadb190a/cmd/operator-sdk/new/cmd.go#L149) above (can't comment on that line):
```Go
	cfg := &input.Config{
		Repo:           filepath.Join(projutil.CheckAndGetProjectGoPkg(), projectName),
		AbsProjectPath: filepath.Join(projutil.MustGetwd(), projectName),
		ProjectName:    projectName,
	}
```

We're using `projutil.CheckAndGetProjectGoPkg()` to get the Repo name. Which would return the error `Project not in $GOPATH` from [`MustSetGoPath()`](https://github.com/operator-framework/operator-sdk/blob/01b6886ae9324212ca05b20a75cf98d0dadb190a/internal/util/projutil/project_util.go#L195-L197)